docs/markdown/announcements.md

```diff
@@ -0,0 +1,65 @@
+The announcements feature is used to notify users of new content. Read more about the business usage of this feature in [the KB article](https://kb.wisc.edu/myuw/page.php?id=63903). The following documentation talks more about the implementation details of the announcements.
+
+![./img/announcement-character.png](./img/announcement-character.png)
+
+## The Theme
+By default, if you set nothing in [the theme](#/md/theming) the character in the image above will show. You can of course provide the `mascotImg` variable in the theme to override this.
+
+## The Image
+The image itself should have the following characteristics (see what I did there?):
+
+<div class='row'>
+<div class='col-sm-6'>
+<div class='width-100'>
+![./img/mascot/mascot-w-comments.png](./img/mascot/mascot-w-comments.png)
+</div>
+</div>
+<div class='col-sm-6'>
+<ul>
+<li>A) height : 40px</li>
+<li>B) width : 60px</li>
+<li>C) Transparent Background</li>
+<li>D) Eye height should be 12px down, so that you can see it during the hidden state.</li>
+<li>Can be an animated gif, but animations should be limited to blinking eyes.</li>
+<li>If you do have an arm, suggest the top of it to be under the top 16px.</li>
```

Comment:
Thoughts on removing this suggestion, since the default has an arm that's doesn't follow this suggestion, or maybe we could ask @keirserrie for an updated default robot with a smaller hand.